src/Components/test/E2ETest/ServerExecutionTests/LocalizationTest.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BasicTestApp;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests
+{
+    // For now this is limited to server-side execution because we don't have the ability to set the
+    // culture in client-side Blazor.
```

Comment:
I'd have thought that setting the culture for client-side Blazor isn't too troublesome, but we would certainly have issues with loading the satellite assemblies from the `.resx` files.

Reply:
You could do it in Program.Main (user code) or in the JS code that boots up mono. I spent a little time looking at it but didn't really dig in. If we think it's cheap I'd really like to be able to get it working, but it's also going to require some API design I think.